composer.json

```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^7.3 || ^8.0",
+        "illuminate/macroable": "^8.16",
```

Comment:
Can we copy this code? So we don't depend from it?

Reply:
Sure, it's just a single trait. 👍🏻 If we are doing it inline, it should be fine to always return `$this`, and then we can simplify to:
```php
Expectation::macro('toBeResource', function () {
    Assert::assertIsResource($this->value);
});
```